symplyphysics/laws/thermodynamics/heat_transfer_equation.py

```diff
@@ -0,0 +1,48 @@
+from sympy import (Eq, solve)
+from symplyphysics import (units, Quantity, Symbol, print_expression, validate_input,
+    validate_output)
+
+# Description
+# heat flow calculation Ф = λ * (T1-T2)/ d * S
+# where:
+# λ = material_thermal_conductivity (Watts/(K*m))
+# T1 = inside_temperature (C')
+# T2 = outside_temperature (C')
+# d = wall_thickness (m)
+# S = wall_area (m2)
+# Note: heat flow (Ф) is measured in Watts
+# 
+# This formula describes Fourier's law for heat transfer and is used to calculate heat flow (Ф) through a material or structure.
+# The Fourier formula is used to determine the amount of heat that is transferred through a material from one side to the other due to the material's thermal conductivity, temperature gradient, material thickness, and cross-sectional area.
```

Comment:
Add explanation for heat flow sign. Positive heat flow is for flow from inside to outside? Or it's vice versa?